factory/cryptoSigningParams.go

```diff
@@ -12,29 +12,29 @@ import (
 	"github.com/ElrondNetwork/elrond-go/data/state"
 )
 
-type cryptoSigningParamsFactory struct {
+type cryptoSigningParamsLoader struct {
```

Comment:
This will be merged with the crypto signing factory.

Reply:
done

Reply:
done